src/components/Home/Home.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Home: React.FC = () => {
```

Comment:
Apparently [this pattern is discouraged against](https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components)